src/store/modules/app.js

```diff
@@ -51,6 +51,7 @@ const steno_keyboards = ['gergo', 'georgi'];
 const getters = {
   firmwareFile: state => state.firmwareFile,
   filter: state => state.filter,
+  keyboard: state => state.keyboard,
```

Comment:
don't do this please. Use mapState and just get it directly. Getters should only be used when the value needs to be changed in flight, all these getters probably need to go away.